components/collection/EditModal.vue

```diff
@@ -125,6 +125,56 @@
             </div>
           </div>
         </NeoField>
+
+        <NeoField
+          :label="$t('mint.collection.permission.label')"
+        >
+          <div class="w-full flex flex-col gap-4">
+            <div class="flex items-center justify-between">
+              <p>
+                {{ $t('mint.mintType') }}
+              </p>
+              <NeoSelect
+                v-model="selectedMintingType"
+              >
+                <option
+                  v-for="menu in COLLECTION_MINTING_TYPES_OPTIONS"
+                  :key="menu.value"
+                  :value="menu.value"
+                >
+                  {{ menu.text }}
+                </option>
+              </NeoSelect>
+            </div>
+
+            <div>
+              <div class="flex justify-between">
+                <p>{{ $t('mint.collection.permission.noPriceSet') }}</p>
```

Comment:
after setting a price this did not change 

![CleanShot 2024-12-06 at 14 36 43@2x](https://github.com/user-attachments/assets/a7c33327-8faf-4ced-9fd3-9b527f64343f)

Reply:
refresh the page could help, but I will also add a refetch after signing

Reply:
Updated. This text will be changed after toggling.